src/vision/status_quo/alan_runs_into_stack_trouble.md

```diff
@@ -0,0 +1,60 @@
+# 😱 Status quo stories: Alan runs into stack allocation trouble
+
+[Alan]: ../characters/alan.md
+[Grace]: ../characters/grace.md
+[Niklaus]: ../characters/niklaus.md
+[Barbara]: ../characters/barbara.md
+
+[Alan runs into stack allocation trouble and is able to fix problems]: TODO
+
+### The problem
+
+Alan is working on a networking service written using async Rust and the tokio async runtime. One day, after many changes to his code base, Alan runs his application and hits an error:
```

Comment:
I've been pondering this intro -- I was originally thinking that it was good to be specific, but it also seems like the fact that this is a networking service and using tokio is really not relevant at all. Maybe we should just say:

```suggestion
One day, as Alan is working on his async Rutst project, he runs his application and hits an error:
```